src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf

```diff
@@ -278,6 +278,22 @@
                 <source>This value should not be identical to {{ compared_value_type }} {{ compared_value }}.</source>
                 <target>Giá trị không được phép giống như {{ compared_value_type }} {{ compared_value }}.</target>
             </trans-unit>
+            <trans-unit id="85">
```

Comment:
Should be 87, and incremented for there (all languages must have the same ids).

Reply:
done